### PR TITLE
Add @Override annotation to missing methods

### LINKED ISSUE #1075: Add @Override annotations to implementations
We have been received similar kinds of issues which points out that there's an implementation but interface is missing.

Maybe we can prevent it by adding @Override annotations to all of implementations as JedisCluster does it now.

### PR DESCRIPTION
**Pull request by marcosnils**
Fixes #1075.

Signed-off-by: Marcos Lilljedahl <marcosnils@gmail.com>

### CODE REVIEW

**Comment by marcosnils** [comment]
This PR will be included from 3.0 on wards it's quite big to make down merge it.

File: src/main/java/redis/clients/jedis/Jedis.java [hunk 1]
```diff
@@ -2894,33 +3055,28 @@ public Long pttl(final String key) {
...
   }
 
+  @Override
   public String set(final String key, final String value, final String nxxx) {
     checkIsInMultiOrPipeline();
     client.set(key, value, nxxx);
     return client.getStatusCodeReply();
   }
 
-  public String set(final String key, final String value, final String nxxx, final String expx,
```
Comment (HeartSaVioR) [inline src/main/java/redis/clients/jedis/Jedis.java]: @marcosnils :+1: Great work catching this!

**Comment by HeartSaVioR** [comment]
@marcosnils 
I'm wondering we can backport the changes on interfaces to 2.7 and 2.8 to expand the interfaces earlier.
Actually it is missed thing, which can be handled as a bug.
What do you think?

**Comment by HeartSaVioR** [comment]
Forgot to tell you did a amazing work! :100:

**Comment by HeartSaVioR** [comment]
@marcosnils Never mind. I will take care of it.

**Comment by sazzad16** [comment]
@HeartSaVioR @marcosnils I took a stab and downmerged this!

**Pull request merged**